.github/workflows/pr_check_webapp_dotnet_windows.yml

```diff
@@ -71,14 +71,14 @@ jobs:
         npm install
         npm run build
     
-    - name: Azure authentication
-      uses: azure/login@v1
-      with:
-        creds: ${{ secrets.AZURE_WEBAPP_SPN }}
+    # - name: Azure authentication
+    #   uses: azure/login@v1
+    #   with:
```

Comment:
Ideally SPN is the preferred mechanism for auth, any reason to move to publish profile?

Reply:
Lets discuss this offline.